### PR TITLE
Use the AAL atlas to filter streamlines

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

File: AFQ/segmentation.py [hunk 1]
```diff
@@ -42,7 +43,9 @@ def __init__(self,
                  b0_threshold=0,
                  prob_threshold=0,
                  rng=None,
-                 return_idx=False):
+                 return_idx=False,
+                 filter_by_endpoints=True,
+                 dist_to_aal=4):
```
Comment (arokem) [inline AFQ/segmentation.py]: @jyeatman : in your experience, what is a reasonable distance to use for filtering? Using this distance still seems to leave quite a few streamlines that look spurious (at least for CST, at least in the example dataset).

Comment (jyeatman) [reply AFQ/segmentation.py]: I would have suggested 4mm just looking at fiber endpoints and atlas voxels. The atlas is very rough and just serves to clear up the really bad streamlines

Comment (arokem) [reply AFQ/segmentation.py]: Great - that helped - calibrating to the voxel size, so the distance is in mm, solved the particular issue I was seeing with that CST, so 4 mm does seem to be a good way to go (but 8 mm is too much 😄)

**Comment by arokem** [comment]
OK - I've been playing around with this all day yesterday, and I'm pretty sure that it's doing the right thing. 

In particular, I think that this resolves both #168, as well as #174 (resolved in https://github.com/yeatmanlab/pyAFQ/commit/b4f1a7a493d6204cded47235965d978f45c9e672). So I  think that this is ready to merge.

**Pull request merged**